web/spec/supabase.yml

```diff
@@ -777,7 +777,6 @@ pages:
 
   auth.api.createUser():
```

Comment:
@J0 @dthyresson  thoughts on how we can go about explaining that this method overwrites the authentication settings defined in the project via the dashboard?